packages/wrangler/src/index.tsx

```diff
@@ -1330,6 +1338,10 @@ export async function main(argv: string[]): Promise<void> {
           default: false,
           describe:
             "If a log would have been filtered out, send it through anyway alongside the filter which would have blocked it.",
+        })
+        .option("legacy-env", {
+          type: "boolean",
```

Comment:
```suggestion
          type: "boolean",
          hidden: true,
```

Let's not show it in the menu at all right now, across all commands

Reply:
Sounds good, I will make all the legacy-env options hidden and update the tests accordingly.